plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -61,30 +49,34 @@ export const CatalogPage = ({
   initiallySelectedFilter = 'owned',
   columns,
   actions,
-}: CatalogPageProps) => {
-  const styles = useStyles();
-
-  return (
-    <CatalogLayout>
-      <Content>
-        <ContentHeader title="Components">
-          <CreateComponentButton />
-          <SupportButton>All your software catalog entities</SupportButton>
-        </ContentHeader>
-        <div className={styles.contentWrapper}>
-          <EntityListProvider>
-            <div>
+}: CatalogPageProps) => (
+  <CatalogLayout>
+    <Content>
+      <ContentHeader title="Components">
+        <CreateComponentButton />
+        <SupportButton>All your software catalog entities</SupportButton>
+      </ContentHeader>
+      <Grid container spacing={2}>
+        <EntityListProvider>
+          <Grid container item sm={12} lg={2} alignContent="flex-start">
```

Comment:
I'm not sure it's valid to have container and item on the same node. I've been seeing problems with that in the past.

There's some double spacing going on at the edges of the page, and I think this may be the reason.

![Screenshot 2021-07-06 at 13 53 13](https://user-images.githubusercontent.com/3097461/124595798-c0a27b80-de61-11eb-815d-5590593c69e1.png)

Reply:
Yes - it was exactly as you said. 

<img width="382" alt="Screenshot 2021-07-06 at 17 16 03" src="https://user-images.githubusercontent.com/8904624/124625490-23a20b80-de7e-11eb-8b55-aab7a02f5ea0.png">